opentelemetry/src/main/scala/zio/telemetry/opentelemetry/Tracing.scala

```diff
@@ -132,7 +136,7 @@ object Tracing {
     for {
       nanoSeconds <- currentNanos
       span        <- getCurrentSpan
-    } yield span.addEvent(name, attributes.asJava, nanoSeconds)
+    } yield span.addEvent(name, buildAttributes(attributes), nanoSeconds)
```

Comment:
Arguably we could choose to directly use opentelemetry's `Attributes` as an argument here, and not create our own intermediate data type (`Map`)

Reply:
Sounds good.

Reply:
Updated https://github.com/zio/zio-telemetry/pull/189/commits/2cd3eea88dfdc18bb5d346d2b1165755dbfc3afb